engine/src/star_system.cpp

```diff
@@ -422,8 +423,13 @@ void NebulaUpdate( StarSystem *ss )
     }
 }
 
-void GameStarSystem::createBackground( StarSystem::StarXML *xml )
+void GameStarSystem::createBackground( Star_XML *xml )
 {
+    cout << "bgname " << xml->backgroundname << endl;
+    cout << "numstars " << xml->numstars << endl;
+    cout << "backgroundDegamma " << xml->backgroundDegamma << endl;
+    cout << "numnearstars " << xml->numnearstars << endl;
+    cout << "starsp " << xml->starsp << endl;
```

Comment:
These should go to the boost log assuming they're even needed still.

Reply:
@royfalk You should be able to just replace `cout` with `BOOST_LOG_TRIVIAL(debug)`, and then take off the ` << endl` at the end.

Reply:
Deleted.